homeworks/vazgen_harutyunyan/Homework_10/QueueTest.java

```diff
@@ -0,0 +1,50 @@
+class Queue {
+    int[] array = new int[10];
+    int position = 0;
+    int firstIndex = 0;
+    int firstElement;
+
+
+    void push(int a) {
+        if(position < array.length) {
+            array[position++] = a;
+        } else {
+            System.out.println("Your queue has already full");
+        }
```

Comment:
this doesn't seem complete, what if you push 10 items, then pop 2 and call push again?